src/test/resources/alma-fix/990110881770206441.json

```diff
@@ -85,10 +85,11 @@
     "focus" : {
       "id" : "http://www.wikidata.org/entity/Q6210",
       "label" : "Kreis Coesfeld",
-      "type" : [ "http://www.wikidata.org/entity/Q106658" ],
+      "type" : [ "\"http://www.wikidata.org/entity/Q106517174" ],
+      "@geoData" : " http://www.wikidata.org/entity/Q20738811\"",
```

Comment:
@dr0i This results because of the csv sep_char: `,`. MF does not know `"` as a protected value sign.